### ISSUE TITLE
sentry crash when uploading sourcemaps

### ISSUE THREAD

**Issue by hawkrives** [StoDevX/AAO-React-Native#3907]
https://app.circleci.com/jobs/github/StoDevX/AAO-React-Native/61963

![image](https://user-images.githubusercontent.com/464441/62238102-e8e0f000-b397-11e9-9b9e-032448532f1a.png)

```
[01:45:34]: $ npx @sentry/cli releases files $(PRODUCT_BUNDLE_IDENTIFIER)-2.8.0 upload-sourcemaps --dist 65213 --strip-prefix /Users/distiller/project --rewrite ../ios-release.bundle.map
[01:45:34]: ▸ sh: PRODUCT_BUNDLE_IDENTIFIER: command not found
[01:45:36]: ▸ error: Found argument '-2' which wasn't expected, or isn't valid in this context
[01:45:36]: ▸ USAGE:
[01:45:36]: ▸ sentry-cli releases files <VERSION> <SUBCOMMAND>
[01:45:36]: ▸ For more information try --help
sh: PRODUCT_BUNDLE_IDENTIFIER: command not found
error: Found argument '-2' which wasn't expected, or isn't valid in this context
```

I think we just need double quotes around the `PRODUCT_BUNDLE_IDENTIFIER` part.

**Comment by rye** [comment]
Yeah, this is an issue preventing nightlies from being successful.

`$(PRODUCT_BUNDLE_IDENTIFIER)` is coming from `fastlane`… this command is built as https://github.com/StoDevX/AAO-React-Native/blob/282b34db58c210eea30e5bab087ee94a69535124/fastlane/lib/sourcemaps.rb#L61-L71 which specifically calls out to the `bundle_identifier` method. And this method is defined as https://github.com/StoDevX/AAO-React-Native/blob/282b34db58c210eea30e5bab087ee94a69535124/fastlane/lib/sourcemaps.rb#L47-L55
which indicates to me that the contents of the key `CFBundleIdentifier` in `ios/AllAboutOlaf/Info.plist` at the time that these are run is exactly the string `$(PRODUCT_BUNDLE_IDENTIFIER)`—this is _exactly_ what is currently in that file.

So… is there a better way to get what gets substituted by `$(PRODUCT_BUNDLE_IDENTIFIER)`? Is this file _not_ overwritten by `fastlane`?

**Comment by hawkrives** [comment]
Xcode does the substitution of $() variables in the Info.plist…

**Comment by rye** [comment]
Okay, so how do we get the value it substitutes?

**Comment by hawkrives** [comment]
This is closed by #4155 … even if sentry still fails to upload, this particular issue has been fixed.
